src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverStatsExporter.cs

```diff
@@ -88,6 +88,11 @@ internal class StackdriverStatsExporter
             displayNamePrefix = GetDisplayNamePrefix(configuration.MetricNamePrefix);
         }
 
+        static StackdriverStatsExporter()
+        {
+            USER_AGENT = $"opencensus-csharp/{Assembly.GetExecutingAssembly().GetName().Version.ToString()}";
```

Comment:
please add `try{}catch` so missing reflection permissions will not crash the process